File: Database/src/main/java/sep3/database/Model/UserList.java
package sep3.database.Model;

import java.util.ArrayList;

public class UserList {

    private ArrayList<User> users;

    public UserList() {
        users = new ArrayList<>();
    }

    public UserList(ArrayList<User> users) {
        this.users = users;
    }
    public void addUser(User user)
    {
        users.add(user);
    }
    public void removeUser(User user)
    {
        users.remove(user);
    }

    public ArrayList<User> getUsers() {
        return users;
    }

    public void setUsers(ArrayList<User> users) {
        this.users = users;
    }
}

File: Application/Application/Models/UserList.cs
using System.Collections.Generic;


namespace Application.Models
{
    public class UserList
    {
        public List<User> Users { get; set; }

        public UserList()
        {
            this.Users = new List<User>();
        }

        public void AddUser(User user)
        {
            Users.Add(user);
        }

        public void RemoveUser(User user)
        {
            Users.Remove(user);
        }
    }
}